_includes/scripts.html

```diff
@@ -0,0 +1,53 @@
+<!-- dynamically determine scripts needed for this config -->
+{% assign _scripts = '' | split: '' %}
+{% assign _async_marker = 'uswds_async=true' %}
+{% assign _site_scripts = site.scripts %}
+{% unless _site_scripts -%}
+  {% assign _uswds_js = '/assets/uswds/js/uswds.min.js' | append: '?' | append: _async_marker %}
+  {% assign _site_scripts = '' | split: '' | push: _uswds_js %}
+  {% if site.anchor_js_targets %}
```

Comment:
Since we're no longer supporting multiple sites with this theme, we could probably simplify this entire file, which would make it less error-prone to update in the future. It's definitely not worth holding up the PR for, though! Maybe worth opening a ticket about for the future.